### PR TITLE
issue707: returning null instead of empty json object 

### PR DESCRIPTION
**Pull request by anvk**
for empty XML nodes

I'm not sure how to write tests for this pull request so I would appreciate any help or suggestions on how to do so.

### CODE REVIEW

**Comment by jsdevel** [comment]
See `request-response-samples`.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "soap",
-  "version": "0.9.3",
+  "version": "0.9.4",
```
Comment (jsdevel) [inline package.json]: we'll handle version bumping.  Please remove this.

**Comment by anvk** [comment]
I removed version bump per request and changed some of the tests to accommodate that change. Although I have problems with tests of such format:

```
<?xml version="1.0" encoding="UTF-8"?>
<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
   <soap:Header />
   <soap:Body>
      <Response xmlns="http://www.example.com/v1" />
   </soap:Body>
</soap:Envelope>
```

What would be the result of this? I assume maybe `{}` would make more sense than `null` but again not entirely sure....

**Comment by jsdevel** [comment]
see the other response samples under `test/`

**Comment by jsdevel** [comment]
@anvk can you update the tests?

**Comment by jsdevel** [comment]
Taking this up in https://github.com/vpulim/node-soap/pull/784.

**Pull request closed without merging**